hw3/01-characters.js

```diff
@@ -1,4 +1,63 @@
 // url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+const url = "https://thronesapi.com/api/v2/Characters/rgdr";
 
-fetch(url);
+const charContainer = document.querySelector("#characters-container");
+charContainer.classList.add("card-deck");
```

Comment:
Why not just declare this class "card-deck" directly as an attribute on the element in the html file? Why does this class have to be added using javascript?